api/v1alpha1/lvmvolumegroupnodestatus_types.go

```diff
@@ -22,6 +22,13 @@ import (
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+type DeviceDiscoveryPolicy string
+
+const (
+	DeviceDiscoveryPolicyPreConfigured  DeviceDiscoveryPolicy = "PreConfigured"
```

Comment:
These constants are lacking documentation when not navigating from DeviceDiscoveryPolicy docs, maybbe move the docs from DeviceDiscoveryPolicy and write something more here?

Reply:
I think Preconfigured is one word, so you dont need to capitalise configured.